src/DeviceInterface/InterfacePrototype.js

```diff
@@ -230,14 +233,36 @@ class InterfacePrototype {
 
         await this.setupAutofill()
         await this.refreshSettings()
+
+        // this is the temporary measure to support windows whilst we still have 'setupAutofill'
+        // eventually all interfaces will use this
+        if (!this.isEnabledViaSettings()) {
+            return
+        }
+
         await this.setupSettingsPage()
         await this.postInit()
 
         if (this.settings.featureToggles.credentials_saving) {
-            listenForGlobalFormSubmission(this.scanner.forms)
+            formSubmissionsApi(this.scanner.forms)
         }
     }
 
+    /**
+     * This is to aid the migration to all platforms using Settings.enabled.
```

Comment:
Can you tell me more about this `Settings.enabled`? Is this a single setting in the UI? On macOS autofill can be turned off by type, there's no central switch to turn all autofill off.